utils/src/test/kotlin/ArchiveUtilsTest.kt

```diff
@@ -76,5 +77,43 @@ class ArchiveUtilsTest : StringSpec() {
             fileB.exists() shouldBe true
             fileB.readText() shouldBe "b\n"
         }
+
+        "Debian deb archive can be unpacked" {
+            val archive = File("src/test/assets/testpkg.deb")
+
+            archive.unpack(outputDir)
+
+            val extractedScriptFile = outputDir.resolve("usr/bin/test")
+            extractedScriptFile.exists() shouldBe true
+
+            val extractedControlFile = outputDir.resolve("control")
+            extractedControlFile.exists() shouldBe true
+            val expectedControl = File("src/test/assets/control-expected.txt").readText()
+            extractedControlFile.readText() shouldBe expectedControl
+
+            listOf("data.tar.xz", "control.tar.xz").forEach { tarFileName ->
+                val tarFile = outputDir.resolve(tarFileName)
+                tarFile.exists() shouldBe false
+            }
+        }
+
+        "Debian udeb archive can be unpacked" {
+            val archive = File("src/test/assets/testpkg.udeb")
```

Comment:
Instead of adding two separate identical assets, how about making a copy of the file in the test to eliminate the redundancy?

Also,  IIUC only one test is needed and for the other format it would suffice to mock the return value of the archive function.

Reply:
Done